src/App.tsx

```diff
@@ -26,70 +27,120 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
+  visibleGoods.sort((firstGood, secondGood) => {
+    switch (sortType) {
+      case SortType.ALPHABET:
+        return firstGood.localeCompare(secondGood);
+
+      case SortType.LENGTH:
+        return firstGood.length - secondGood.length;
+
+      default:
+        return 0;
+    }
+  });
```

Comment:
Don't start sorting if you don't have sort type + don't check the sort type inside the iterator, check it before

```suggestion
  switch (sortType) {
    case SortType.ALPHABET: 
      visibleGoods.sort((firstGood, secondGood) => (
        firstGood.localeCompare(secondGood);
      )
    
    case .....
```